docs/activation-strategies.md

```diff
@@ -45,23 +45,23 @@ Gradually activates feature toggle. Stickiness based on session id. It is almost
 
 ## gradualRolloutRandom
 
-Randomly activates the feature toggle. No stickiness. We have found this rollout strategy very useful in some scenarios, especially when we enable a feature which is not visible to the user. It is also the strategy we use to sample metrics and error reports.
+This gradual rollout random strategy randomly activates a feature toggle and has no stickiness. We have found this rollout strategy very useful in some scenarios, especially when we enable a feature which is not visible to the user. It is also the strategy we use to sample metrics and error reports.
```

Comment:
gradual rollout random strategy => `GradualRolloutRandom` strategy in order to be consistent?

Reply:
Yup, consistency is good!